src/main/java/practice/StreamPractice.java

```diff
@@ -50,7 +70,11 @@ public List<Person> selectMenByAge(List<Person> peopleList, int fromAge, int toA
      */
     public List<Person> getWorkablePeople(int fromAge, int femaleToAge,
                                           int maleToAge, List<Person> peopleList) {
-        return Collections.emptyList();
+        return peopleList.stream()
+                .filter(e -> e.getAge() >= fromAge && ((e.getAge() <= maleToAge
+                        && e.getSex() == Person.Sex.MAN) || (e.getAge() <= femaleToAge
+                        && e.getSex() == Person.Sex.WOMAN)))
```

Comment:
```suggestion
                .filter(p -> p.getAge() >= fromAge
                        && (p.getSex().equals(Person.Sex.MAN) && p.getAge() <= maleToAge
                        || p.getSex().equals(Person.Sex.WOMAN) && p.getAge() <= femaleToAge))
```
just for better readability